packages/botonic-react/src/components/custom-message.jsx

```diff
@@ -1,15 +1,21 @@
 import React from 'react'
+import { INPUT } from '@botonic/core'
+import merge from 'lodash.merge'
+
 import { Message } from './message'
 import { Reply } from './reply'
-import { INPUT } from '@botonic/core'
+import { mapObjectNonBooleanValues } from '../utils'
```

Comment:
if you create a util folder for deprecation logs, you might create a dom.js with mapObjectNonBooleanValues

Reply:
will do it in another pr, i was thinking about creating it under `/utils/object.js`